packages/bits/src/lib/sorter/sorter.component.ts

```diff
@@ -193,10 +207,18 @@ export class SorterComponent implements OnChanges, OnInit, OnDestroy, AfterViewI
     }
 
     public ngOnDestroy() {
+        this.menuKeyControlListeners.forEach(listener => listener());
         this.onDestroy$.next();
         this.onDestroy$.complete();
     }
 
+    public toggleSorterMenu(): void {
+        if (!this.overlay.showing) {
+            this.sorterKeyControlService.announceDropdown(true);
```

Comment:
If the menu overlay with items is hidden (`!this.overlay.showing`) you won't be able to get the menu items length during the announcement, will you?